internal/plugin/connectors/http/generic/config.go

```diff
@@ -34,34 +35,35 @@ func (c *config) validate(credsByID connector.CredentialValuesByID) error {
 	return nil
 }
 
-// renderedHeaders returns the config's header templates filled in with the
+// Render returns the config's `config` section templates filled in with the
 // given credentialValues.
-func (c *config) renderedHeaders(
+func render(
```

Comment:
Hmm... why was this changed from being a method on config object?

Reply:
I think we need a better name than just `render` - maybe we should add one more word to describe what it's actually rendering.

Reply:
- Before, the only thing it needed from `config` was access to the Header object. Since we now pass those objects in as a parameter, it seemed unnecessary to keep it as an object specific method. I don't mind either way.
- Agreed on the naming, I can make it more specific